core/templates/dev/head/services/contextual/UrlServiceSpec.js

```diff
@@ -30,18 +30,9 @@ describe('Url Service', function() {
   beforeEach(module('oppia'));
   beforeEach(inject(function($injector) {
     UrlService = $injector.get('UrlService');
-    spyOn(UrlService, 'queryUrlParams').and.returnValue(window);
+    spyOn(UrlService, 'getCurrentLocation').and.returnValue(window);
```

Comment:
I think you've got the varname wrong here: window !== location. Maybe call the var above something like mockLocation. That also makes it clearer that it's a mock.

Reply:
Done